datamodel/src/test/resources/actionable/ActionableCharacteristics.37.tsv

```diff
@@ -1,3 +1,3 @@
-type	cutoffType	cutoff	source	sourceEvent	sourceUrls	nctId	title	acronym	genderCriterium	countriesAndCities	hospitalsPerCity	treatment	treatmentApproachesDrugClass	treatmentApproachesTherapy	applicableCancerType	applicableDoid	blacklistCancerTypes	level	approvalStatus	direction	evidenceUrls
-MICROSATELLITE_UNSTABLE	EQUAL_OR_GREATER	4	CKB_EVIDENCE	msi	http							Nivolumab			All cancer types	X	Hematologic cancer;2531	A	GUIDELINE	RESPONSIVE	http1,http2
-MICROSATELLITE_UNSTABLE			CKB_TRIAL	msi	http	NCT1	study1		both	Netherlands(Rotterdam;Eindhoven),Belgium(Brussel)	Rotterdam(EMC),Eindhoven(Catharina),Brussel(UZ Brussel)	Pembro,Immuno			All cancer types	X	Hematologic cancer;2531	B	GUIDELINE	RESPONSIVE	http3,http4
\ No newline at end of file
+type	cutoffType	cutoff	source	date	sourceEvent	sourceUrls	nctId	title	acronym	genderCriterium	countriesAndCities	hospitalsPerCity	treatment	treatmentApproachesDrugClass	treatmentApproachesTherapy	applicableCancerType	applicableDoid	blacklistCancerTypes	description	level	approvalStatus	direction	evidenceUrls
+MICROSATELLITE_UNSTABLE	EQUAL_OR_GREATER	4	CKB_EVIDENCE	2021-02-03	msi	http							Nivolumab			All cancer types	X	Hematologic cancer;2531	efficacy evidence	A	GUIDELINE	RESPONSIVE	http1,http2
+MICROSATELLITE_UNSTABLE			CKB_TRIAL	1970-01-01	msi	http	NCT1	study1		both	Netherlands(Rotterdam;Eindhoven),Belgium(Brussel)	Rotterdam(EMC),Eindhoven(Catharina),Brussel(UZ Brussel)	Pembro,Immuno			All cancer types	X	Hematologic cancer;2531		B	GUIDELINE	RESPONSIVE	http3,http4
```

Comment:
See comment above. Don't think it's preferred to have 1970-01-01 in the output files. We could set this to an empty string?

Reply:
Ok, though this looks suspiciously like handling a nullable field 😄. Anyway I'll update, but how should the db dao handle this? I've made the field non-null sql DATETIME field there and it would be populated with the EPOCH value. Should this be nullable in mysql? Or prefer empty strings to DATETIME fields there?

Reply:
I think I would keep in the 1970 date rather than empty. Don't think we should do specific serialisation / deserialisation logic. As for the SQL model, I think I would write 1970 as well since we should do as-little-as-possible conversion (the SQL data should simply be a view on the data)

Reply:
Ok, let's just leave it this way :)